### PR TITLE
using GET method for searching

### PR DESCRIPTION
**Pull request by alexey-klimuk**
using GET method for searching

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF17] search via GET and POST requests so that you can copy the search results url](https://trello.com/c/JXoxPQja/1628-tariff17-search-via-get-and-post-requests-so-that-you-can-copy-the-search-results-url)

File: app/views/search/search.html.erb [hunk 1]
```diff
@@ -42,3 +42,6 @@
     </div>
   <% end %>
 </article>
+<script type="application/javascript">
+  window.history.pushState({},"", "<%= url_for params.except(:authenticity_token, :_method, :controller, :action) %>".replace(/&amp;/g, '&'));
```
Comment (matthewford) [inline app/views/search/search.html.erb]: we do full page caching, I'm not sure this will work,

Comment (matthewford) [inline app/views/search/search.html.erb]: I think we need to do it server side when we do the redirect

Comment (alexey-klimuk) [inline app/views/search/search.html.erb]: @matthewford I didn't find anything except this solution

Comment (alexey-klimuk) [inline app/views/search/search.html.erb]: @matthewford  we rendering search.html.erb, but yeah we can also add params to redirects

Comment (alexey-klimuk) [inline app/views/search/search.html.erb]: BTW @matthewford params are already in redirect url 
https://github.com/bitzesty/trade-tariff-frontend/blob/master/app/controllers/search_controller.rb#L15
https://github.com/bitzesty/trade-tariff-frontend/blob/master/app/controllers/search_controller.rb#L10

**Comment by matthewford** [comment]
@alexey-klimuk lets do as we originally though and convert this to a GET call

**Comment by alexey-klimuk** [comment]
@matthewford converted to GET

**Pull request merged**